docs/logs/archiving.md

```diff
@@ -0,0 +1,146 @@
+title: Log Archiving
+description: Archiving logs to an S3-compatible object store
+
+<div style="padding-top:40px;display:grid;grid-template-columns:150px 150px 150px 150px;grid-template-rows:100px 100px;">
+  <a href="https://aws.amazon.com/s3" target="_blank" title="Amazon AWS S3">
+    <img src="../../images/logs/archiving/aws-logo.png" style="height:80px;width:134px;margin-top:auto;">
+  </a>
+  <a href="https://www.digitalocean.com/products/spaces" target="_blank" title="Digital Ocean Spaces">
+    <img src="../../images/logs/archiving/digital-ocean-logo.png" style="height:80px;width:80px;margin-top:auto;">
+  </a>
+  <a href="https://www.ibm.com/cloud/object-storage" target="_blank" title="IBM Cloud Object Storage">
+    <img src="../../images/logs/archiving/ibm-logo.png" style="height:80px;width:90px;margin-top:auto;">
+  </a>
+  <a href="https://azure.microsoft.com/services/storage" target="_blank" title="Azure Cloud Storage">
+    <img src="../../images/logs/archiving/azure-logo.png" style="height:80px;width:104px;margin-top:auto;">
+  </a>
+  <a href="https://cloud.google.com/storage" target="_blank" title="Google Cloud Storage" style="grid-column-start:2">
+    <img src="../../images/logs/archiving/google-logo.png" style="height:80px;width:99px;margin-top:auto;">
+  </a>
+  <a href="https://min.io" target="_blank" title="MinIO Object Storage" style="align-self:center;">
+    <img src="../../images/logs/archiving/minio-logo.png" style="height:30px;width:150px;margin-top:auto;">
+  </a>
+</div>
+
+You can configure a Logs App to forward all the logs it receives
+to an S3-compatible object store.
+
+### How do I configure archiving for my Logs App?
+
+In Sematext web app, go to `Integrations > Apps` view
```

Comment:
In Sematext web app, go to `Integrations > Apps` view

=>

In Sematext Cloud, go to the `Integrations > Apps` view

Reply:
I wanted to write that, but then remembered we also have on-prem.

Reply:
Aha, then we can do _Sematext Cloud, or Enterprise,_.

Reply:
That is why we/I tend to use just "Sematext".  I also believe that is really how people end up calling products - by the company name: Newrelic, Datadog, Splunk, etc. are all examples.  Splunk does have on-prem and cloud, but everyone just says Splunk unless they really want to get very specific.  So just "Sematext" in docs.